lldb/test/API/python_api/address_range/main.cpp

```diff
@@ -0,0 +1,3 @@
+#include <iostream>
+
+int main() { std::cout << "Hello World!" << std::endl; }
```

Comment:
No need to use <iostream> here right? I would just do a simpler `main`:
```
int main() {
  return 0;
}
```
(remove `#include` and use of `std::cout`). This will reduce the size and speed up compilation.